{"name": "test_case[case3]", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <test_case.Test_Case object at 0x000001DA928C3C88>\ncase = {'case_description': 'post /topic_collect/de_collect 取消主题', 'case_expect': '{\"success\":false}', 'case_method': 'post', 'case_params': '', ...}\n\n    @pytest.mark.parametrize('case',Excel_Handler().get_excel_data())\n    def test_case(self,case):\n>       response = RequestHandler(case).get_response()\nE       TypeError: 'tuple' object is not callable\n\nscripts\\test_case.py:13: TypeError"}, "parameters": [{"name": "case", "value": "{'case_project': 'cnodejs项目', 'case_description': 'post /topic_collect/de_collect 取消主题', 'case_url': 'https://cnodejs.org/api/v1//topic_collect/de_collect', 'case_method': 'post', 'case_params': '', 'case_expect': '{\"success\":false}', 'other': '咱们没有用户 accessToken，所以返回为false就算对'}"}], "start": 1574774734212, "stop": 1574774734790, "uuid": "4a121e9d-81c1-4579-b7c9-da2cd4500188", "historyId": "c01cd7b8eb6761254f7350d3c46bef6b", "testCaseId": "89a27520489a549f0caeaffa7948244d", "fullName": "scripts.test_case.Test_Case#test_case", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "Test_Case"}, {"name": "host", "value": "IG6I7Y1F5I7TZH5"}, {"name": "thread", "value": "14484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}